lib/modules/datasource/conan/common.ts

```diff
@@ -8,3 +8,12 @@ export const conanDatasourceRegex = regEx(
   /(?<name>[a-z\-_0-9]+)\/(?<version>[^@/\n]+)(?<userChannel>@\S+\/\S+)/,
   'gim'
 );
+
+export function getRevision(packageName: string): string | undefined {
```

Comment:
```suggestion
export function getRevision(packageName: string): string | null {
```

prefer `null` ofer `undefined`